.github/workflows/ci.yaml

```diff
@@ -0,0 +1,48 @@
+name: Node.js CI
+
+on:
+  pull_request:
+    branches: [ "develop" ]
+
+jobs:
```

Comment:
These jobs will be running in seperate containers and you seem not to have connected them in a logical order. Either put all of them in one single job, or make the jobs dependent on each other in a logical way. I mean if the linting fails, there is no benefit in making the testing continue, right?

Reply:
it should be outdated as well